pkgs/applications/kde/kleopatra.nix

```diff
@@ -1,21 +1,31 @@
 {
-  mkDerivation, lib, kdepimTeam,
+  mkDerivation, fetchpatch, lib, kdepimTeam,
   extra-cmake-modules, kdoctools,
   boost, gpgme, kcmutils, kdbusaddons, kiconthemes, kitemmodels, kmime,
   knotifications, kwindowsystem, kxmlgui, libkleo, kcrash
 }:
 
 mkDerivation {
   pname = "kleopatra";
+
+  patches = [
+    (fetchpatch {
```

Comment:
It's not in any way critical, but typically we either name the patch using the name attribute like this `name = "fix_build.patch";` or put a comment to show why we are patching.

Reply:
I totally forgot. The reason is that the released version is not compatible with the version of pgpme we (and arch) have.